### PR TITLE
Exercise collections (Playlists)

### PR DESCRIPTION
**Pull request by swemail**
<image src="https://user-images.githubusercontent.com/1139207/221870972-32364f66-3f98-4022-bfd7-adc796dcdb40.png" width="200" /> <image src="https://user-images.githubusercontent.com/1139207/221871393-39ffdc37-8a28-4692-a0ed-6cc088e08bb6.png" width="200" /> <image src="https://user-images.githubusercontent.com/1139207/221871816-2679da71-48bb-4ef2-b3d5-803980a1e370.png" width="200" /> <image src="https://user-images.githubusercontent.com/1139207/222096570-420ba6a7-e8d2-4c4e-9b57-810248a7ae86.png" width="200" />

### CODE REVIEW

File: client/src/lib/components/Cards/CollectionCards/CollectionListCard.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    [collection],
+  );
+
+  const onPress = useCallback(() => {
+    navigate('Collection', {collectionId: collection.id});
+  }, [navigate, collection]);
+
+  return (
+    <Container onPress={onPress}>
+      <Display16>{collection.name + 'sdfsdfsdf dsfdsfsdf'}</Display16>
```
Comment (gewfy) [inline client/src/lib/components/Cards/CollectionCards/CollectionListCard.tsx:62]: Debug ;)

File: client/src/lib/components/Cards/CollectionCards/CollectionListCard.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <GraphicsWrapper>
+          <Image source={image} />
+        </GraphicsWrapper>
+      </ImageContainer>
+      <Spacer8 />
+    </Container>
+  );
+};
+
+export default CollectionCard;
```
Comment (gewfy) [inline client/src/lib/components/Cards/CollectionCards/CollectionListCard.tsx:74]: These new cards are not very re-usable. I think you either split them into a layout and container components. Or move the closer to it's usage.

File: client/src/lib/user/state/state.ts [hunk 3]
```diff
@@ -24,8 +24,14 @@ type PinnedSession = {
...
 };
 
+export type SavedCollection = {
+  id: string;
+  statedAt: string;
+};
+
 export type UserState = {
   pinnedSessions?: Array<PinnedSession>;
+  savedCollections?: Array<SavedCollection>;
```
Comment (gewfy) [inline client/src/lib/user/state/state.ts:34]: Is it really saved? Isn't it pinned or started?

File: content/src/collections/f82c2201-b35a-47f0-a2ad-26895e3fa0b6.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  "es": {
+    "id": "f82c2201-b35a-47f0-a2ad-26895e3fa0b6",
+    "exercises": [
+      "095f9642-73b6-4c9a-ae9a-ea7dea7363f5",
+      "5b0dea9f-c665-41ce-a779-601d0cfa433d",
+      "3188a5ed-a1d6-451c-ae2a-f44f4df37495"
+    ]
+  }
+}
```
Comment (gewfy) [inline content/src/collections/f82c2201-b35a-47f0-a2ad-26895e3fa0b6.json:38]: Should we maybe remove these test collections for now?

File: client/src/lib/components/Cards/CollectionCards/CollectionListCard.tsx (hunk 2 above)
Comment (gewfy) [reply client/src/lib/components/Cards/CollectionCards/CollectionListCard.tsx:74]: Maybe move them into `routes/screens/Sessions` for now?

Comment (gewfy) [reply client/src/lib/components/Cards/CollectionCards/CollectionListCard.tsx:74]: Aaaand if you decide to make them re-usable. Please add them to UI library too.

**Review by gewfy — APPROVED** [review]

File: client/src/lib/user/state/state.ts (hunk 3 above)
Comment (transient-jonas) [reply client/src/lib/user/state/state.ts:34]: enrolledCollections?

File: content/src/collections/f82c2201-b35a-47f0-a2ad-26895e3fa0b6.json (hunk 4 above)
Comment (swemail) [reply content/src/collections/f82c2201-b35a-47f0-a2ad-26895e3fa0b6.json:38]: Yes, they was just for me testing, will remove them. Totally forgot that too 😂

File: client/src/lib/user/state/state.ts (hunk 3 above)
Comment (swemail) [reply client/src/lib/user/state/state.ts:34]: pinned perhaps, consistent with where pinned sessions end up kind of

**Pull request merged**